src/main/webapp/app/exercises/text/assess-new/text-assessment-area/text-assessment-area.component.ts

```diff
@@ -18,20 +19,28 @@ export class TextAssessmentAreaComponent implements OnChanges {
     @Input() submission: TextSubmission;
     @Input() textBlockRefs: TextBlockRef[];
     @Input() readOnly: boolean;
+    @Input() selectedFeedbackIdWithConflicts?: number;
+    @Input() conflictMode: boolean;
```

Comment:
This is not needed. I think this can be replaced everywhere by `feedbackConflicts  === undefined`?

Reply:
So in the text-submission-assessment.component you might have multiple feedback with conflicts. You select one of them to see its conflicts. `selectedFeedbackIdWithConflicts ` indicates which feedback with conflicts is selected.